### PR TITLE
Bootstrap redesign

### PR DESCRIPTION
**Pull request by YoshiRulz**
My goal with this was to change as little as possible from Bootstrap's defaults.

Looks like this in desktop Firefox:
Screenshot_20221010_084749
(also looks good in Android Firefox)

### CODE REVIEW

File: src/css/index.scss [hunk 1]
```diff
@@ -1,3 +1,92 @@
+//@use "sass:map";
+
+// 1. Include functions first (so you can manipulate colors, SVGs, calc, etc)
+@import "../../node_modules/bootstrap/scss/functions";
```
Comment (Lego6245) [inline src/css/index.scss]: I think there must be a way to do these imports such that parcel can bundle in bootstrap without the absolute reference to node modules here.

File: src/css/index.scss [hunk 2]
```diff
@@ -1,3 +1,92 @@
...
+}
+
+#the_table {
+    max-width: 10cm;
+
+    tbody {
+        counter-reset: rowNumber;
+
+        tr {
+//            &:nth-child(0n+1) {
```
Comment (Lego6245) [inline src/css/index.scss]: The goal is to color the first 3, right? Does nth-child(1) not work? (etc).

**Review by Lego6245 — CHANGES_REQUESTED** [review]
This looks great and a real simple improvement. Couple things to potentially change to keep things clean.

File: src/css/index.scss (hunk 2 above)
Comment (YoshiRulz) [reply src/css/index.scss]: I tried several variations of that and couldn't get it working. You're welcome to try.

**Comment by Lego6245** [comment]
I think I got something working in pure css, works on my copy of Edge.

**Comment by YoshiRulz** [comment]
Can probably reduce lock file diff noise too but I don't know how.

**Pull request closed without merging**